cli/args.go

```diff
@@ -149,6 +149,8 @@ func parseTerragruntOptionsFromArgs(terragruntVersion string, args []string, wri
 		return nil, err
 	}
 
+	strictMode := parseBooleanArg(args, OPT_TERRAGRUNT_STRICT, false)
```

Comment:
Can you also rename the vars + attributes used in golang for this to also mention `STRICT_VALIDATE`/`validateStrictMode`? That way we won't get confused when working with the code.

Reply:
Roger! - done in e3d0b82

Reply:
Just took a look at e3d0b829b35d5bea8057328fdf3e5fe2a7300536. Don't forget to rename `opts.Strict` too!

Reply:
Thanks! Renamed in 2f59cd7